### PR TITLE
[WIP] Feature/candidate election aggregate

### PR DESCRIPTION
**Pull request by jmcarp**
Show election aggregates on candidate detail pages.

* [x] Implement election and cycle select
* [x] Optionally aggregate candidate totals
* [x] Optionally aggregate candidate other spending
* [x] Write unit tests for new code

Depends on https://github.com/18F/openFEC/pull/1352.

### CODE REVIEW

File: static/templates/electionCycles.hbs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<fieldset class="js-subcycle-select toggles">
+  {{#each this}}
+  <label>
+    <input
+        type="radio"
+        name="cycle-toggle"
+        value="{{this.max}}"
+        class="panel-toggle-control"
```
Comment (noahmanger) [inline static/templates/electionCycles.hbs]: The inputs don't need a class.

File: static/templates/electionCycles.hbs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<fieldset class="js-subcycle-select toggles">
+  {{#each this}}
```
Comment (noahmanger) [inline static/templates/electionCycles.hbs]: Small thing, but this should have a legend right after fieldset:
`<legend class="label">Time period:</legend>`

File: static/templates/electionCycles.hbs (hunk 1 above)
Comment (jmcarp) [inline static/templates/electionCycles.hbs]: Oh, I modeled the markup after the toggles on the committee detail page. Do we not need that class there either? I can get rid of them all if they don't do anything.

Comment (jmcarp) [inline static/templates/electionCycles.hbs]: Never mind, got it.

Comment (noahmanger) [inline static/templates/electionCycles.hbs]: Yeah looks like that's for the js so should be prefixed with `.js-`.

File: static/templates/electionCycles.hbs (hunk 2 above)
Comment (noahmanger) [inline static/templates/electionCycles.hbs]: Yep.

**Pull request merged**